packages/modules/pricing/integration-tests/__tests__/services/pricing-module/price-set.spec.ts

```diff
@@ -422,6 +437,40 @@ moduleIntegrationTestRunner({
               ],
             })
           )
+
+          const [priceRules] = await service.listPriceRules({
+            price_set_id: [priceSet.id],
+          })
+
+          expect(eventBusEmitSpy.mock.calls[0][0]).toHaveLength(3)
+          expect(eventBusEmitSpy.mock.calls[0][0][0]).toEqual(
+            composeMessage(PricingEvents.price_set_created, {
+              service: Modules.PRICING,
+              action: CommonEvents.CREATED,
+              object: "price_set",
+              data: { id: priceSet.id },
+            })
+          )
+
+          expect(eventBusEmitSpy.mock.calls[0][0][1]).toEqual(
+            composeMessage(PricingEvents.price_created, {
+              service: Modules.PRICING,
+              action: CommonEvents.CREATED,
+              object: "price",
+              data: { id: priceSet.prices![0].id },
+            })
+          )
+
+          expect(eventBusEmitSpy.mock.calls[0][0][2]).toEqual(
+            composeMessage(PricingEvents.price_rule_created, {
+              service: Modules.PRICING,
+              action: CommonEvents.CREATED,
+              object: "price_rule",
+              data: {
+                id: priceRules.id,
+              },
+            })
+          )
```

Comment:
todo: we need to add some expectation in the test bellow, I can see that we jumped to some tests later on